appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java

```diff
@@ -567,6 +568,14 @@ private void setANameProperty(String aname) {
           SettingsConstants.YOUNG_ANDROID_SETTINGS_APP_NAME, aname);
     }
   }
+  
+  private void setTitleVisibleProperty(String visible) {
+    if (visible.equals("True")) {
+      titleBar.setVisible(true);
+    } else {
+      titleBar.setVisible(false);
+    }
+  }
```

Comment:
There are better/cleaner ways to write this. You should do `titleBar.setVisible(visible);` instead.

Reply:
@conorshipp, this is true. I think I will instead use `Boolean.parseBoolean(visible);`. Also, please first test your ideas, before making expectations that you can directly set visibility. :wink: